Makefile

```diff
@@ -185,6 +185,18 @@ helm.generate:
 	./hack/helm.generate.sh $(BUNDLE_DIR) $(HELM_DIR)
 	@$(OK) Finished generating helm chart files
 
+helm.test: helm.generate
+	@helm unittest --file tests/*.yaml --file 'tests/**/*.yaml' deploy/charts/external-secrets/
```

Comment:
It would be good to add the unittest installation to our developer guide, as I sense this is going to be part of our make-reviewable criteria in the long term